src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java

```diff
@@ -118,32 +119,54 @@ public class JavadocUtil {
                     + "environment variable using -Xms:<size> and -Xmx:<size>.";
 
     /**
-     * Method that removes the invalid directories in the specified directories. <b>Note</b>: All elements in
-     * <code>dirs</code> could be an absolute or relative against the project's base directory <code>String</code> path.
+     * Method that removes invalid classpath elements in the specified paths. <b>Note</b>: All elements in
+     * <code>paths</code> could be absolute or relative against the project's base directory <code>String</code> path.
+     * When pruning class paths, you can optionally include "*.jar" files in the result, otherwise only directories are
+     * permitted.
      *
      * @param project the current Maven project not null
-     * @param dirs the collection of <code>String</code> directories path that will be validated.
-     * @return a List of valid <code>String</code> directories absolute paths.
+     * @param paths the collection of <code>String</code> paths that will be validated
+     * @param includeJars whether to include JAR files in the result
+     * @return a list of valid <code>String</code> directories and optionally JAR files as absolute paths
```

Comment:
what does valid mean here?

Reply:
Why are you asking questions about existing comments in your product, which I have just copied/moved and slightly adjusted? If it was good enough so far, it should be good enough now.

Reply:
There was no prunePaths method here before. If you're adding a public method, clear documentation is required.

Reply:
Look, I understand if you are very busy and do not have enough time to triage bugs or review code. But if you cannot see the difference between a new method and a renamed, slightly extended one to convey the more generic behaviour, differentiating it by name from the original method the code was factored out from, I really cannot help you. I can live without you consenting to and merging these two bugfixes, if you want to coerce me to do your job to refactor the existing documentation, after I already did your job of fixing two old bugs. Merge it, fine. Refactor the docs after the merge, be my guest. Or don't merge it, hurting yourself and your users, also fine. I am done here.